Formula/ignition-sensors4.rb

```diff
@@ -1,28 +1,18 @@
 class IgnitionSensors4 < Formula
   desc "Sensors library for robotics applications"
   homepage "https://github.com/ignitionrobotics/ign-sensors"
-  url "https://github.com/ignitionrobotics/ign-sensors/archive/766301b738d9a92a7b17fbdefdbc0f0487fb1bea.tar.gz"
-  version "3.999.999~0~20200408~84f8018"
-  sha256 "1d5334a25c9cb34c36744d1d2ebdc7dd0c92c8bf214df4eabc26f487493058b6"
-  revision 1
-
   head "https://github.com/ignitionrobotics/ign-sensors", :branch => "master"
 
-  bottle do
-    root_url "https://osrf-distributions.s3.amazonaws.com/bottles-simulation"
-    sha256 "b67923ae883156046ec75e7641d06a0554b25779f01623d4e5e3390224d02953" => :mojave
-  end
-
   depends_on "cmake" => [:build, :test]
   depends_on "pkg-config" => [:build, :test]
 
   depends_on "ignition-cmake2"
   depends_on "ignition-common3"
   depends_on "ignition-math6"
-  depends_on "ignition-msgs5"
+  depends_on "ignition-msgs6"
   depends_on "ignition-rendering4"
-  depends_on "ignition-transport8"
-  depends_on "sdformat9"
+  depends_on "ignition-transport9"
+  depends_on "sdformat10"
```

Comment:
Broke the changes to this file into https://github.com/osrf/homebrew-simulation/pull/1069, with an updated source code commit that expects these dependencies.